catalyst/dl/registry.py

```diff
@@ -24,6 +24,12 @@ def _transforms_loader(r: Registry):
     try:
         import albumentations as m
         r.add_from_module(m)
+
+        from albumentations import pytorch as p
+        r.add_from_module(p)
```

Comment:
what to you think about adding `albu` prefix?

Reply:
or, maybe `A` for Albumentations and `C` for Catalyst?

Reply:
We do not support any other augmentation library (I think, we won't).

And I think it's unnecessary to add a prefix to all transforms in config

Reply:
About, `C` for Catalyst. Users will not use our contrib transforms very often, I prefer to use full form, it is much readable and clean

Reply:
there are 2 ways:
1. we add prefix for all our sources for transformations, except Core Catalyst one
2. we don't use any prefixes at all... but it could cause name collision
all other variants looks unscalable

PS. And yup, we will support more transformations source in the future

Reply:
> we add a prefix for all our sources for transformations, except Core Catalyst one
why is that? `albumentations` has a lot more common transforms than in catalyst.core

it's totally okay to use prefixes for some unusual classes or methods

Reply:
> PS. And yup, we will support more transformations source in the future

nice, and for them, i would favor adding a prefix making `albumentations` preferred one

Reply:
look at a typical transform config
https://github.com/catalyst-team/catalyst/pull/600/files#diff-e8e0a33e0d5dfbfe552d3a0fc812d1bdR5

adding an `albu.` prefix for every single transform is long and unnecessary

Reply:
`but making albumentations preferred one, without prefix` why?

Reply:
ODS support

Reply:
the main point of this PR is friendship between albumentations, SMP and maybe ttach in future

Reply:
I dont want to have any possible bias to any other framework (except Pytorch, of course),
I like ODS, nevertheless, we need to be open for any other framework on the same rules.

Reply:
you can use any prefix, even `A` is good enough, 
or add some global prefix-source for all transformations

Reply:
ok, i've added `A.` prefix